wpiunits/src/main/java/edu/wpi/first/units/Units.java

```diff
@@ -261,6 +261,10 @@ private Units() {
   public static final Dimensionless Percent =
       derive(Value).splitInto(100).named("Percent").symbol("%").make();
 
+  // Frequency
+  /** The unit of frequency. */
+  public static final Velocity<Dimensionless> Hertz = Value.per(Second);
```

Comment:
This will need naming and aliasing. It'll currently appear as `"<?> per Second"` and `"<?>/s"` for the name and symbol, respectively